decentraland/unity-explorer
# fix: Mute world/avatar/chat sounds while we are in auth or loading screens

## What does this PR change?
Fix #746 

- Creation of `AudioMixerVolumesController` for managing audio mixer volumes. It exposes the nest methods:
  - `MuteGroup(AudioMixerExposedParam groupParam)`
  - `UnmuteGroup(AudioMixerExposedParam groupParam)`

## How to test the changes?
1. Launch the explorer
2. While you're in the Authentication screen, check that you only only listen to the music and not the world, avatar or chat sounds.
3. While you're in any loading screen, check that you only only listen to the music and not the world, avatar or chat sounds.
4. Check that, once you're inside the world and controlling your avatar, all the sounds are recovered to the original volume.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
